packages/babel-types/src/modifications/flow/removeTypeDuplicates.ts

```diff
@@ -17,7 +17,7 @@ function getQualifiedName(node: t.GenericTypeAnnotation["id"]) {
  * Dedupe type annotations.
  */
 export default function removeTypeDuplicates(
-  nodes: ReadonlyArray<t.FlowType | false | null | undefined>,
+  nodes: (t.FlowType | false | null | undefined)[],
```

Comment:
This is exposed as part of the public API, so we cannot change this.

We could clone `nodes` at the beginning of the function so that we can then `.push()` into it, but we would need a few benchmarks to check if it's worth it.

Reply:
In this case benchmarks are not that useful because we should evaluate a number of different cases.